boxsdk/object/comment.py

```diff
@@ -23,13 +31,10 @@ def reply(self, message):
             `unicode`
         """
         url = self.get_type_url()
-        message_type = 'tagged_message' if '@[' in message else 'message'
-        data = {
-            message_type: message,
-            'item': {
-                'type': 'comment',
-                'id': self.object_id
-            }
+        data = Comment.construct_params_from_message(message)
```

Comment:
This should be `self.construct_params_from_message`.

staticmethods and classmethods can be called with `self`. Python will do the correct thing.

Hard-coding the current class should almost always be avoided, since it prevents overriding the behavior in a subclass.

Another thing that would work (but isn't necessary) is `self.__class__.construct_params_from_message`, but it shouldn't be any different from `self.construct_params_from_message`.